pkg/gen/input/makefile/internal/file/windows-code-signing.sh.template

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+
+{{ .Header }}
+
```

Comment:
Unsure if this script would benefit from `set -e` (http://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin) to make the script exit immediately on any error.

Unsure if this exist in plain /bin/sh